faucet/faucet_dot1x.py

```diff
@@ -144,124 +141,87 @@ def set_mac_str(self, valve, valve_index, port_num):
         self.mac_to_port[mac_str] = (valve, port)
         return mac_str
 
-    def port_up(self, valve, dot1x_port):
-        """Setup the dot1x forward port acls.
+    def nfv_sw_port_up(self, dp_id, dot1x_ports, nfv_sw_port, acl_manager):
+        """Setup the dot1x forward port acls when the nfv_sw_port comes up.
         Args:
-            dot1x_port:
-            valve:
+            dp_id (int):
+            dot1x_ports (Iterable of Port objects):
+            nfv_sw_port (Port):
+            acl_manager (ValveAclManager):
 
         Returns:
             list of flowmods
         """
-        self.dot1x_speaker.port_down(get_mac_str(self.dp_id_to_valve_index[valve.dp.dp_id],
-                                                 dot1x_port.number))
-        nfv_sw_port = valve.dp.dot1x['nfv_sw_port']
+        self.dot1x_speaker.port_down(
```

Comment:
(This (port_down()) was an error on my part and should have been port_up(), think @MichaelWasher has fixed it in pr #2697
But seeing how this PR is not adding functionality, I'm guessing someone will refactor later.)

Thinking about when the NFV port goes up.
The system is effectively down, we miss logoff/new authentication attempts coming from the supplicants but should still get port events via OpenFlow.

i think there are a few options we could do:

A) Don't notify Chewie as it does not currently do anything if the NFV port is down.
B) Notify Chewie, and later we can do something (e.g. Send a preemptive Identity Request). We could interpret the NFV port being down as meaning that everyone is deauthenticated, but I'm not sure if we would want to do this.

I think option A is fine for the moment - so this line should be able to be removed, and it shouldn't affect functionality.